bindings/python/.evergreen/test.sh

```diff
@@ -10,6 +10,8 @@ set -o errexit  # Exit the script with error if any of the commands fail
 
 # MONGOCRYPT_DIR is set by libmongocrypt/.evergreen/config.yml
 MONGOCRYPT_DIR="$MONGOCRYPT_DIR"
+git clone git@github.com:mongodb-labs/drivers-evergreen-tools.git
+sed -i 's/deterministic=True//g' drivers-evergreen-tools/.evergreen/mongodl.py || true
```

Comment:
We need this because the `deterministic` parameter was only added in Python 3.8, and the newest version of Python available on RHEL70 is Python 3.6. https://docs.python.org/3/library/sqlite3.html#sqlite3.Connection.create_function

Reply:
And the `|| true` bit is needed because this will fail on macOS because of a quirk of the `-i` flag. It needs an empty string first for it to do it in-place: `sed -i '' 's/deterministic=True//g' drivers-evergreen-tools/.evergreen/mongodl.py`. However, because we don't need the path on macOS, because we have newer pythons available, it does not matter.

Reply:
Please open a PR to remove deterministic=True from https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/mongodl.py

Reply:
Can this line be removed now that the PR was merged?

Reply:
Removed.